draft-ietf-tls-exported-authenticator.md

```diff
@@ -511,6 +511,13 @@ The signatures generated with this API cover the context string
 "Exported Authenticator" and therefore cannot be transplanted into other
 protocols.
 
+In TLS 1.3 the client can not explicitly learn from the TLS layer whether its
+Finished message was accepted. Because the application traffic keys are not
+dependent on the client's final flight, the client cannot learn whether the
+server ever received it.  To avoid disagreement between the client and server
```

Comment:
```suggestion
Finished message was accepted. Because the application traffic keys are not
dependent on the client's final flight, receiving messages from the server
does not prove that the server received the client's Finished. To avoid disagreement between the client and server
```